### DISCUSSION TITLE
blog/threads-in-c-language

### DISCUSSION THREAD

**Discussion by giscus** [elskow/helmyl-pages#5] [Q&A]
# blog/threads-in-c-language

Explore the concept of threads in the C programming language, their use cases, and how they enable parallel execution for improved performance.

http://localhost:3001/blog/threads-in-c-language

**Comment by elskow** [comment]
```rs
fn main() {
   println!("hello world");
}
```

**Reply by elskow** [reply]
hello world

**Reply by elskow** [reply]
hello from localhost
